modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/exec/rel/TableScanNodeExecutionTest.java

```diff
@@ -263,7 +263,8 @@ private static class TestInternalTableImpl extends InternalTableImpl {
         ) {
             super(
                     QualifiedNameHelper.fromNormalized(SqlCommon.DEFAULT_SCHEMA_NAME, "test"),
-                    1,
+                    2, // zone id.
```

Comment:
What do you think if we extract the values as constant fields with the corresponding names? Commentaries are better then nothing, but any test class may be evolved and commentary may be erased and so on.

Reply:
Ok. Fixed.